src/main/java/edu/greenblitz/tobyDetermined/subsystems/Gyros/GyroFactory.java

```diff
@@ -0,0 +1,20 @@
+package edu.greenblitz.tobyDetermined.subsystems.Gyros;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+
+public class GyroFactory {
+
+    public static IGyro create(){
+        switch (RobotMap.ROBOT_TYPE) {
+            case FRANKENSTEIN:
```

Comment:
What happens if there is no Gyro?

Reply:
you make a simulation gyro (the default), and if hte robot dont have a gyro we have a little problem ;)